host_applications/linux/apps/gencmd/gencmd.c

```diff
@@ -36,6 +36,18 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 #include "interface/vmcs_host/vc_vchi_gencmd.h"
 
+void show_usage()
+{
+   printf( "Usage: vcgencmd [-t] [COMMAND]\n" );
```

Comment:
Only a very minor point, but I wonder if `vcgencmd [-t] [commands|COMMAND]` might be better here? :man_shrugging: